modules/helm-oci-prereqs.adoc

```diff
@@ -8,13 +8,13 @@ $ sudo cp rootCA.pem   /etc/pki/ca-trust/source/anchors/
 $ sudo update-ca-trust extract
 ----
 
-* **Experimental feature:** Many of the commands for interacting with Helm and OCI registries make use of the `helm chart` subcommand.  At the time of writing, OCI support in Helm is still marked as an “experimental” feature and must be enabled explicitly. This is accomplished by setting the environment variable `HELM_EXPERIMENTAL_OCI=1`.
+* **Generally available:** As of Helm 3.8, OCI registry support for charts is now generally available. Many of the commands for interacting with Helm and OCI registries make use of the `helm chart` subcommand.
 
-* **Install Helm client:** Download your desired version from link:https://github.com/helm/helm/releases[], for example, link:https://get.helm.sh/helm-v3.5.3-linux-amd64.tar.gz[]. Unpack it and move the helm binary to its desired destination:
+* **Install Helm client:** Download your desired version from the link:https://github.com/helm/helm/releases[Helm releases] page. Unpack it and move the helm binary to its desired destination:
 +
 ----
-$ tar -zxvf helm-v3.5.3-linux-amd64.tar.gz
+$ tar -zxvf helm-v3.0.0-linux-amd64.tar.gz
```

Comment:
why the installer change to 3.0, suppose should be version 3.8 , like "helm-v3.8.2-linux-amd64.tar.gz"